MsGraphicsPkg/Library/SimpleUIToolKit/SimpleUIToolKit.inf

```diff
@@ -57,4 +57,3 @@
 
 [BuildOptions]
   *_VS2013x86_*_CC_FLAGS       = /Oi /Og /fp:fast /arch:SSE
```

Comment:
And you want to keep VS2103 why?

Reply:
What's the new normalized VS version?  Since these flags are scope to a VS compiler but suitably generic, can I just use MSFT:*_*_*_CC_FLAGS (will that work for any VS version?)?